pom.xml

```diff
@@ -176,7 +177,7 @@
         <dependency>
             <groupId>org.apache.sling</groupId>
             <artifactId>org.apache.sling.api</artifactId>
-            <version>2.16.4</version>
+            <version>2.18.2</version>
```

Comment:
Do you use updated API really?

Reply:
Yes, I do. ResourceProvider.AUTH_CLONE was introduced in 2.18.2, specifically to support the original change this PR is trying to fix.